packages/fuel-gauge/src/predicate/utils/predicate/assertBalances.ts

```diff
@@ -1,8 +1,8 @@
 import { bn, toHex } from 'fuels';
-import type { BN, BigNumberish, WalletLocked } from 'fuels';
+import type { BN, BigNumberish, WalletUnlocked } from 'fuels';
 
 export const assertBalances = async (
-  receiver: WalletLocked,
+  receiver: WalletUnlocked,
```

Comment:
Because `Wallet.generate` returns an `Unlocked` wallet and the test utility is [primarily used](https://github.com/FuelLabs/fuels-ts/blob/mc/chore/integrate-launch-test-node/packages/fuel-gauge/src/predicate/predicate-arguments.test.ts#L45-L60) in that context